File: TriathlonCrossPlatform/Traithlon.Model/Referee.cs
namespace Triathlon.Model
{
    public class Referee : Entity
    {
        public new int Id { get => base.Id; set => base.Id = value; }

        public string Name { get; set; }
        public string Password { get; set; }
        public int IdEvent { get; set; }

        public Referee(int id, string name, string password, int idEvent) : base(id)
        {
            this.Name = name;
            this.Password = password;
            this.IdEvent = idEvent;
        }
    }
}

File: TriathlonServerJava/Common/src/main/java/ro/mpp/triathlon/model/Referee.java
package ro.mpp.triathlon.model;

//import model.Entity;

public class Referee extends Entity {
    private String name;
    private String password;
    private Integer id_event;

    public Referee(int id, String name, String password, Integer id_event) {
        super(id);
        this.name = name;
        this.password = password;
        this.id_event = id_event;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public Integer getIdEvent() {
        return id_event;
    }

    public void setIdEvent(int id_event) {
        this.id_event = id_event;
    }
}
